reliability-engineering/dockerfiles/govsvc/aws-terraform/Dockerfile

```diff
@@ -0,0 +1,17 @@
+ARG ubuntu_digest
```

Comment:
Just want to note that this is different to the way we build other images in this pipeline - we just build `FROM ubuntu:latest` and rely on `load_bases` to set the version of ubuntu appropriately.  Your way is more explicit but also quite a bit more plumbing code. Also we now have two different patterns for doing the same thing and it'd be nice to be consistent.

Reply:
yeah there the main difference here is that this automates the building of the image as each new version of terraform appears (rather than having to go and bump the number manually).

This was partially because I needed to rebuild several different versions not just the latest one, and didn't want to have to change the value in several places.

pinning the ubuntu digest only really adds that the "re-run-with-same-inputs" works as expected... other than that it's a bit pointless